Library/Homebrew/version.rb

```diff
@@ -13,6 +13,7 @@ class Version
 
   include Comparable
 
+  # @api public
```

Comment:
If it's public: might want to document it too?

Reply:
Thanks; added documentation.